akka/classic/src/test/scala/net/ruippeixotog/akka/testkit/specs2/MySpec.scala

```diff
@@ -12,39 +13,39 @@ class MySpec extends AkkaSpecification {
 
     "receive messages" in {
       testActor ! "hello" // expect any message
-      this must receiveMessage
+      (this: TestKitBase) must receiveMessage
```

Comment:
Hmm, why did you have to type this explicitly? Can we avoid it?

Worst case we should provide some handy `def selfTestkit: TestKitBase` in `AkkaSpecification` so users can simply write `selfTestkit must receiveMessage`,  but I'd like to see if we can avoid it altogether.

Reply:
In Scala 3, without this I get an error
```
No given instance of type MySpec.this.ToBlock[
[error]        (=>
[error]          org.specs2.matcher.Matcher[
[error]            net.ruippeixotog.akka.testkit.specs2².api.ReceiveMatcher[
[error]              org.apache.pekko.testkit².TestKitBase, String]
[error]          ]
[error]        ) => org.specs2.execute.Result,
[error]      R] was found for parameter t of method in in trait ExtendedBlockDsl.
```

I think it must be related to the changes in type inference and I'm not sure how to go around it..